src/expo.ts

```diff
@@ -108,6 +132,52 @@ export async function projectOwner(cli: CliName = 'expo'): Promise<string> {
   return stdout.trim();
 }
 
+export async function latestUpdates(cli: CliName = 'eas', branch: string): Promise<string> {
+  let stdout = '';
+
+  try {
+    const command = await which(cli);
+    const args = ['update:list', '--branch', branch, '--json'];
+    stdout = (
+      await getExecOutput(command, args, {
+        silent: true,
+      })
+    ).stdout;
+  } catch (error) {
+    throw new Error(`Could not fetch latest updates, reason:\n${error.message | error}`);
+  }
+
+  if (!stdout) {
+    throw new Error(`Could not fetch the update history`);
+  }
+  const result = JSON.parse(stdout.trim()) as UpdateListElement[];
+  if (!Array.isArray(result)) {
+    throw new Error('The result is valid');
+  }
+  return result[0].group;
+}
+
+export async function lastUpdate(cli: CliName = 'eas', branch: string): Promise<Update[]> {
+  const groupId = await latestUpdates(cli, branch);
+  let stdout = '';
+  try {
```

Comment:
a big part of this function and the previous one are similar.
Is it possible to create a common function between this two?